pyrsia_cli/src/main.rs

```diff
@@ -26,6 +27,24 @@ async fn main() {
 
     // checking and preparing responses for each command and arguments
 
+    let (ls, pg, st) = (
```

Comment:
We should avoid custom abbreviations, my brain absolutely got these wrong.

```suggestion
    let (list, ping, status) = (
```

Reply:
These are just variables being used in the code not in the cli command itself, how does it matter?

Reply:
I find them confusing, I did not understand the code reading it twice